src/Featurizers/Components/DocumentStatisticsEstimator.h

```diff
@@ -74,6 +77,21 @@ struct IterRangeComp {
 
 static constexpr char const * const         DocumentStatisticsEstimatorName("DocumentStatisticsEstimator");
 
+/////////////////////////////////////////////////////////////////////////
+///  \struct        FrequencyAndIndexStruct
+///  \brief         This struct is a combination of values of FrequencyMap and 
+///                 IndexMap for the purpose of keys equivelence of two maps
+///
+struct FrequencyAndIndexStruct {
+    std::uint32_t const TermFrequency;                            // Words and the number of documents that it appears in
+    std::uint32_t const Index;                              
+
+    //will cause error if the struct marked deleted after copy
+    //FEATURIZER_MOVE_CONSTRUCTOR_ONLY(FrequencyAndIndexStruct);
```

Comment:
This shouldn't cause an error - there is probably something else going on.

Reply:
you're right, it looks like copy occurs in generating FrequencyAndIndexMap. Added a comment there